tasks/lazyInstallVendor.js

```diff
@@ -0,0 +1,57 @@
+/**
+ #  Install dependencies to run an action
+ #   - e2e: cypress
+ #   - unit test: karma + jasmine + puppeteer
+ #
+ #  We install them only if they are not inside the project
+ #
+ #  Usage:
+ #       <command> : install for both e2e and test
+ #       <command> e2e : install only for e2e
+ #       <command> test : install only for test
+ */
+const execa = require('execa');
+const fs = require('fs');
+
+const args = process.argv.slice(2);
+const type = args[0];
+
+const testDeps = [
+    'karma',
+    'karma-babel-preprocessor',
+    'karma-chrome-launcher',
+    'karma-cli',
+    'karma-coverage',
+    'karma-jasmine',
+    'karma-junit-reporter',
+    'karma-webpack',
+    'jasmine-core@2.8',
+    'puppeteer@1.6',
+];
+const e2eDeps = [
+    'cypress@3.0'
+];
+
+const install = (testFiles = [], deps = []) => {
+    try {
```

Comment:
what do you think of using `existsSync` ?
the we have a fix instead of a try catch.

Also you should convert the current script, not rfr it. Here if we run
`            install(['./node_modules/karma', './node_modules/cypress'], [...testDeps, ...e2eDeps]);`

if karma doesn't exist but we have cyress, it will rerun both. it's useless. It's easier to have a test by type and a feedback/type

Reply:
though its deprecated, but apparently only `fs.exists` is :D

thing with original lazyInstallVendor without argument is, that if you run `npm i`, it will uninstall packages not in package.json. So if running `npm i karma --no-save` then `npm i cypress --no-save`, it will not have cypress installed.

There werent usages of `lazyInstallVendor` without argument, so i even considered to not include that option. Either way, if using --no-save, you need to run single npm install command.

Reply:
Ho, forgot about this one, it's still doing this ? Indeed that's an issue. We will need this command but not for now ;) 
ok so let's keep it as it is :+1: